### PR TITLE
[Console] Improved the explanation of the hasOption() method

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q | A |
| --- | --- |
| Branch? | 2.7 |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #17597 |
| License | MIT |
| Doc PR | - |

### CODE REVIEW

File: src/Symfony/Component/Console/Input/InputDefinition.php [hunk 1]
```diff
@@ -280,7 +280,9 @@ public function getOption($name)
     }
 
     /**
-     * Returns true if an InputOption object exists by name.
+     * Returns true if an InputOption object exists by name. This method can't
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Console/Input/InputDefinition.php]: Do we need a blank line to split the method title from its description?

Comment (javiereguiluz) [inline src/Symfony/Component/Console/Input/InputDefinition.php]: 👍 I did that in 36e470e

**Comment by stof** [comment]
:+1:

**Comment by fabpot** [comment]
Thank you @javiereguiluz.

**Pull request closed without merging**